cloud4all.html

```diff
@@ -0,0 +1,151 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8" />
+  <link href="https://fonts.googleapis.com/css?family=Chivo:900" rel="stylesheet" type="text/css" />
+  <link rel="stylesheet" type="text/css" href="stylesheets/stylesheet.css" media="screen" />
+  <link rel="stylesheet" type="text/css" href="stylesheets/pygment_trac.css" media="screen" />
+  <link rel="stylesheet" type="text/css" href="stylesheets/print.css" media="print" />
+  <!--[if lt IE 9]>
+  <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
+  <![endif]-->
+  <title>Cloud4all Components in the Global Public Inclusive Infrastructure (GPII)</title>
+</head>
+
+<body>
+  <div id="container">
+    <div class="inner">c
+
+      <header>
+        <h1>Cloud4all Components in the Global Public Inclusive Infrastructure (<abbr>GPII</abbr>)</h1>
+      </header>
+
+      <section id="downloads" class="clearfix">
+        <a href="https://github.com/GPII" id="view-on-github" class="button">
+          <span>View on GitHub</span>
+        </a>
+      </section>
+
+      <hr />
+
+      <section id="main_content">
+        <h2>The Cloud4all Project</h2>
+        <p><a href="http://www.cloud4all.info/">Cloud4all</a> 
+          is one of the projects that contributes to the <abbr>GPII</abbr>.
+          The project is funded from the European Union’s Seventh Framework Programme 
+          for research, technological development and demonstration under grant agreement no 289016.
+        </p>
+
+        <p>Several <abbr>GPII</abbr> components developed during the Cloud4all project
+          are hosted outside the <abbr>GPII</abbr> organization on GitHub.
+        </p>
+
+        <h2>Cloud4all Repositories</h2>
+
+        <h3>End-User Solutions</h3>
+        <ul>
+          <li>
+            <a href="https://github.com/GutiX/chrome4cloud/">chrome4cloud</a>:
+            an extension for Google Chrome
+          </li>
+          <li>
+            <a href="https://github.com/GutiX/firefox4cloud">firefox4cloud</a>:
+            an extension for Mozilla Firefox
+          </li>
+          <li>
+            <a href="https://github.com/barcelonadigital/easit4all">EASIT4ALL</a>:
+            accessible way to use social networks, deployed at 
+            <a href="http://easit4all.com/">EASIT4ALL</a>
+          </li>
+          <li>
+            <a href="https://github.com/javihernandez/gpii-gnome-shell-extension">gpii-gnome-shell-extension</a>:
+            GNOME-Shell extension for the <abbr>GPII</abbr> Personalization Framework
+          </li>
+          <li>
+            <a href="https://build.opensuse.org/project/show/home:javihernandez:GPII:Factory">Factory repository of the <abbr>GPII</abbr> <abbr>RPM</abbr> packages</a>:
```

Comment:
Hey @cstrobbe,

this is the project where last packages are living on:
https://build.opensuse.org/project/show/home:javihernandez:GPII:Factory:Bundle